sql/entities.py

```diff
@@ -23,9 +27,19 @@ class Task(Base):
     date = Column('Date', TIMESTAMP)
     status = Column('Status', Enum('active', 'done', 'deleted', 'archive'),
                     nullable=False, default='active')
-    calendar_date = Column('Calendar_date', DATETIME)
+    calendar_date = Column('Calendar_date', DATETIME, nullable=False,
+                           default='0000-00-00 00:00:00')
+    position = Column('Position', Integer, nullable=False, default=999999)
 
-    def __init__(self, user_id, name, calendar_date):
+    def __init__(self, user_id, name, calendar_date, position=999999):

```

Comment:
Please, change position=999999 to position=None
You must not duplicat logic!!!
This cause should be proccessed by database

Reply:
1) duplicat_e_
2) case, not cause. These are two absolutely different words